modules/tensor_mechanics/include/materials/RadialReturnStressUpdate.h

```diff
@@ -54,25 +74,41 @@ class RadialReturnStressUpdate : public StressUpdateBase
 
 protected:
   virtual void initQpStatefulProperties() override;
-  virtual void computeStressInitialize(Real /*effectiveTrialStress*/,
+
+  /**
+   * Perform any necessary initialization before return mapping iterations
+   * @param effective_trial_stress Effective trial stress
+   * @param elasticityTensor     Elasticity tensor
+   */
+  virtual void computeStressInitialize(Real /*effective_trial_stress*/,
                                        const RankFourTensor & /*elasticity_tensor*/)
   {
   }
-  virtual void iterationInitialize(Real /*scalar*/) {}
-  virtual Real computeResidual(Real /*effectiveTrialStress*/, Real /*scalar*/) { return 0; }
-  virtual Real computeDerivative(Real /*effectiveTrialStress*/, Real /*scalar*/) { return 0; }
-  virtual void iterationFinalize(Real /*scalar*/) {}
+
+  /**
+   * Perform any necessary steps to finalize state after return mapping iterations
+   * @param inelasticStrainIncrement Inelastic strain increment
+   */
   virtual void computeStressFinalize(const RankTwoTensor & /*inelasticStrainIncrement*/) {}
 
-  const unsigned int _max_its;
-  const bool _output_iteration_info;
-  const bool _output_iteration_info_on_error;
-  const Real _relative_tolerance;
-  const Real _absolute_tolerance;
+  virtual Real computeResidual(const Real /*effective_trial_stress*/,
+                               const Real /*scalar*/) override
+  {
+    return 0;
```

Comment:
I'd like to get rid of these method implementations all together, but without the return 0 (which I will move to the source file) I get compiler errors about unimplemented virtual functions.  Is there a way to remove the method implementations without getting the compliler error?